livedata/src/main/java/io/getstream/chat/android/livedata/repository/ChannelRepository.kt

```diff
@@ -66,13 +71,12 @@ class ChannelRepository(var channelDao: ChannelDao, var cacheSize: Int = 100, va
     }
 
     suspend fun retryChannels(): List<ChannelEntity> {
-        val userMap: Map<String, User> = mutableMapOf(currentUser.id to currentUser)
         val channelEntities = selectSyncNeeded()
 
         for (channelEntity in channelEntities) {
-            val channel = channelEntity.toChannel(userMap)
             // TODO: what about channel.members
-            val result = client.createChannel(channel.type, channel.id, channel.extraData).execute()
+            val result =
+                client.createChannel(channelEntity.type, channelEntity.channelId, channelEntity.extraData).execute()
```

Comment:
The stack trace is for a different code path

![image](https://user-images.githubusercontent.com/265409/93375589-9c63a880-f815-11ea-8783-0b121fca1220.png)